### ISSUE TITLE
Conjunction of supine and PP with vera

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Faroese-OFT#9]
What is the least bad thing to do if we have "var" (vera) + Supine og PP? In principle for the var + Supine we want `aux:pass(Supine, var)` but for the PP we would want `cop(PP, var)`. 
  
```
# text = Men í 1986 var landið oyðilagt og á húsagangi.
# text[eng] = But in 1986 the country was ruined and bankrupt.
# labels = to_check ellipsis incomplete
"<Men>"
        "men" CC @cc #1->6
"<í>"
        "í" Pr @case #2->3
"<1986>"
        "1986" Num Dat @obl #3->6
"<var>"
        "vera" V Ind Prt Sg3 @aux:pass #4->6
"<landið>"
        "land" N Neu Sg Nom Def @nsubj #5->6
"<oyðilagt>"
        "oyðileggja" V Sup @root #6->0
"<og>"
        "og" CC @cc #7->9
"<á>"
        "á" Pr @case #8->9
"<húsagangi>"
        "húsagangur" N Msc Sg Dat Indef @conj #9->6
"<.>"
        "." CLB @punct #10->6
```

I have two ideas, maybe there are others:

1. Go with the "most complete structure before the conjunction", that would lead to `aux:pass` because it is the label that the Supine asks for.
2. Go with the globally most coherent "cop-Supn" is more coherent (in that Supn is like an adjective in some ways) than "aux:pass-PP".

**Comment by jnivre** [comment]
Are you sure this is supine? Maybe I am biased by Swedish, but it seems to me that, regardless of whether it is used to adjectivally or passively, it is a past participle (with neuter agreement). The fact that it is coordinated with a PP would then disambiguate it to an adjectival use, and "cop" would be fine.

**Comment by ftyers** [comment]
Good point, at least in [Trond's analyser](http://gtweb.uit.no/cgi-bin/smi/smi.cgi?text=Men+%C3%AD+1986+var+landi%C3%B0+oy%C3%B0ilagt+og+%C3%A1+h%C3%BAsagangi.&action=analyze&lang=fao&plang=eng) only the supine analysis is given. 

But I see what you mean... the verb is type `s20` which gives the perfect/past participle as _oyðilagdur_, which would be masculine, but it doesn't seem to have a neuter form for the past participle for this verb. In any case I'll check this and write again.
